# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/vte/w_exitcode.patch.musl
# Copyright (C) 2022 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- vte-0.70.2/src/widget.cc.vanilla	2022-12-25 18:14:22.057563167 +0100
+++ vte-0.70.2/src/widget.cc	2022-12-25 18:15:33.412556292 +0100
@@ -22,6 +22,10 @@
 
 #include <sys/wait.h> // for W_EXITCODE
 
+#ifndef W_EXITCODE
+#define W_EXITCODE(ret, sig)  ((ret) << 8 | (sig))
+#endif
+
 #include <exception>
 #include <new>
 #include <stdexcept>
